### PR TITLE
test: Initial tests integration

### PR DESCRIPTION
**Pull request by carpe-github**
## This PR
This PR is purposed to add 3 UI test cases in Cypress:
- bridge/cypress/integration/ChangeGitCredentialsTest.js
- bridge/cypress/integration/KeptnSecretAddDeleteTest.js
- bridge/cypress/integration/ProjectCreateTest.js
and related page object classes.

### CODE REVIEW

**Review by ermin-muratovic — APPROVED** [review]
lgtm

**Comment by thisthat** [comment]
Could you please fix the build errors? https://github.com/keptn/keptn/runs/3757034844#step:7:125
After that, we can merge this PR :)

File: bridge/package.json [hunk 1]
```diff
@@ -101,4 +102,4 @@
     "css-what": "~5.0.1",
     "glob-parent": "~5.1.2"
   }
-}
+}
```
Comment (thisthat) [inline bridge/package.json:105]: Please, add always a new line at the end of the file. The same for all files in this PR. :)

**Review by thisthat — COMMENTED** [review]
Please, run `npm install`. You changed the dependency in this file and therefore, you should have a new `package-lock.json` file that is created with the aforementioned command :)

File: bridge/package.json (hunk 1 above)
Comment (carpe-github) [reply bridge/package.json]: The new line is added, thanks for the note!

**Comment by carpe-github** [comment]
> Please, run `npm install`. You changed the dependency in the `package.json` file and therefore, you should have a new `package-lock.json` file that is created with the aforementioned command :)

File: bridge/package.json (hunk 1 above)
Comment (carpe-github) [reply bridge/package.json]: package-lock.json is updated too.

**Review by ermin-muratovic — APPROVED** [review]

**Pull request merged**